rubocop-airbnb/rubocop-airbnb.gemspec

```diff
@@ -25,9 +25,9 @@ Gem::Specification.new do |spec|
     'Gemfile',
   ]
 
-  spec.add_dependency('rubocop', '~> 1.32.0')
-  spec.add_dependency('rubocop-performance', '~> 1.10.2')
-  spec.add_dependency('rubocop-rails', '~> 2.9.1')
-  spec.add_dependency('rubocop-rspec', '~> 2.0.0')
+  spec.add_dependency('rubocop', '~> 1.61.0')
+  spec.add_dependency('rubocop-performance', '~> 1.20.2')
+  spec.add_dependency('rubocop-rails', '~> 2.24.0')
+  spec.add_dependency('rubocop-rspec', '~> 2.26.0')
```

Comment:
I just realized, we should probably remove specification of the patch versions from each of these dependencies (e.g., `~> 1.61` instead of `1.61.0`), to have more flexibility in allowing other minor versions of each dependency.

Reply:
Yeah, sounds good to me, justa added a commit with that change!